model_mommy/mommy.py

```diff
@@ -369,6 +369,9 @@ def instance(self, attrs, _commit):
 
     def _handle_one_to_many(self, instance, attrs):
         for k, v in attrs.items():
+            for value in v:
```

Comment:
This is only a problem on Django 1.9, so it could be moved inside the `if` block below, but I put it here to be consistent.